### PR TITLE
Feature mulitpath integration

### PR DESCRIPTION
**Pull request by Braincrushy**
Added paths to the stages so mutliple ways are possible

### CODE REVIEW

**Comment by Braincrushy** [comment]
TODO make working images? 0_1.png is missing
everything else should work normaly

File: src/os_tester/stages.py [hunk 1]
```diff
@@ -34,30 +52,30 @@ class stages:
     basePath: str
     stagesList: List[stage]
 
-    def __load_stages(self) -> None:
+    def __load_stages(self,yamlFileName: str) -> None:
```
Comment (COM8) [inline src/os_tester/stages.py]:
```suggestion
    def __load_stages(self, yamlFileName: str) -> None:
```

File: src/os_tester/stages.py [hunk 2]
```diff
@@ -34,30 +52,30 @@ class stages:
...
         """
         Loads the stage definition from 'self.basePath' and stores the result inside 'self.stagesList'.
         """
-        ymlFile: str = path.join(self.basePath, "stages.yml")
-        print(f"Loading stages from: {ymlFile}")
+        ymlFilePath: str = path.join(self.basePath,yamlFileName+".yml")
```
Comment (COM8) [inline src/os_tester/stages.py]:
```suggestion
        ymlFilePath: str = path.join(self.basePath, yamlFileName + ".yml")
```

File: src/os_tester/stages.py [hunk 3]
```diff
@@ -1,29 +1,47 @@
...
+    def __init__(self, pathDict: Dict[str, Any], basePath: str):
+        self.checkFile = path.join(basePath, pathDict["check"]["file"])
+        self.checkMseLeq = pathDict["check"]["mse_leq"]
+        self.checkSsimGeq = pathDict["check"]["ssim_geq"]
+        self.actions = pathDict["actions"] if "actions" in pathDict else list()
+        self.nextStage = pathDict["nextStage"]
+
+class stage:
+    """
+    A single stage with the timeout
```
Comment (COM8) [inline src/os_tester/stages.py]:
```suggestion
    A single stage with timeout.
```

File: src/os_tester/stages.py [hunk 4]
```diff
@@ -1,29 +1,47 @@
 import sys
+
 from os import path
 from typing import Any, Dict, List
-
 import yaml  # type: ignore
 
-
-class stage:
+class subpath:
```
Comment (COM8) [inline src/os_tester/stages.py]:
```suggestion
class subPath:
```

Please use lowerCamelCase for class names.

**Review by COM8 — CHANGES_REQUESTED** [review]
Thanks for contributing! Only a few minor changes required and then we are ready to go.

**Review by COM8 — APPROVED** [review]

**Pull request merged**